Library/Homebrew/dependency_collector.rb

```diff
@@ -136,7 +136,7 @@ def parse_class_spec(spec, tags)
   end
 
   def ant_dep(spec, tags)
-    if MacOS.version >= :mavericks
+    if MacOS.version >= :mavericks || !OS.mac?
```

Comment:
No problem! Just wanted to get the conversation started.

Let's be honest, I wasn't expecting this to get merged straightaway.